<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>752</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>40</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="BarraHerramientas">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 85, 255);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>4</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="btnMenu">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>35</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
border:0px;
}

QPushButton:hover
{
border:2px solid #949494;
background-color: #ff0000;
}






</string>
         </property>
         <property name="text">
          <string>  Menu</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>Imajenes e iconos/barra-de-menus.png</normaloff>Imajenes e iconos/barra-de-menus.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>383</width>
           <height>17</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnMinimisar">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>35</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
border:0px;
}

QPushButton:hover
{
border:5px solid #949494;
background-color: #ff0000;
}



</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>Imajenes e iconos/minimizar.png</normaloff>Imajenes e iconos/minimizar.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnRestaurar">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>35</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
border:0px;
}

QPushButton:hover
{
border:5px solid #949494;
background-color: #ff0000;
}


</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>Imajenes e iconos/restaurar.png</normaloff>Imajenes e iconos/restaurar.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnMaximisar">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>35</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
border:0px;
}

QPushButton:hover
{
border:5px solid #949494;
background-color: #ff0000;
}


</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>Imajenes e iconos/maximizar.png</normaloff>Imajenes e iconos/maximizar.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnCerrar">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>35</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
border:0px;
}

QPushButton:hover
{
border:5px solid #949494;
background-color: #ff0000;
}


</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>Imajenes e iconos/cerrar.png</normaloff>Imajenes e iconos/cerrar.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="windows">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="Menu">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>0</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QFrame{
background-color: rgb(159, 159, 159);
}

QPushButton{
background-color: rgb(159, 159, 159);
border-top-left-radius: 20px;
border-bottom-left-radius:20px;
border-top-right-radius: 20px;
border-bottom-right-radius:20px}

QPushButton:hover{
background-color: rgb(255, 255, 255);
border-top-left-radius: 20px;
border-bottom-left-radius:20px;
border-top-right-radius: 20px;
border-bottom-right-radius:20px
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="Inicio">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string>Inicio</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>Imajenes e iconos/pagina-de-inicio.png</normaloff>Imajenes e iconos/pagina-de-inicio.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Software">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string>Software</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>Imajenes e iconos/lenguaje-de-codificacion.png</normaloff>Imajenes e iconos/lenguaje-de-codificacion.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Diseno">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string>Diseño</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>Imajenes e iconos/diseno-grafico.png</normaloff>Imajenes e iconos/diseno-grafico.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Office">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string>Office</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>Imajenes e iconos/office.png</normaloff>Imajenes e iconos/office.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Seguridad">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string>Seguridad</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>Imajenes e iconos/seguridad-informatica.png</normaloff>Imajenes e iconos/seguridad-informatica.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>262</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="Asistente">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>Imajenes e iconos/realidad-virtual.png</normaloff>Imajenes e iconos/realidad-virtual.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="pague">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QStackedWidget" name="Masterpague">
            <property name="currentIndex">
             <number>3</number>
            </property>
            <widget class="QWidget" name="PgInicio">
             <widget class="QPushButton" name="pushButton">
              <property name="geometry">
               <rect>
                <x>120</x>
                <y>140</y>
                <width>291</width>
                <height>121</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(170, 0, 0);</string>
              </property>
              <property name="text">
               <string>PushButton</string>
              </property>
             </widget>
            </widget>
            <widget class="QWidget" name="PgSoftware">
             <widget class="QPushButton" name="pushButton_2">
              <property name="geometry">
               <rect>
                <x>140</x>
                <y>150</y>
                <width>261</width>
                <height>151</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(255, 255, 0);</string>
              </property>
              <property name="text">
               <string>PushButton</string>
              </property>
             </widget>
            </widget>
            <widget class="QWidget" name="PgDiseno">
             <widget class="QPushButton" name="pushButton_3">
              <property name="geometry">
               <rect>
                <x>120</x>
                <y>130</y>
                <width>311</width>
                <height>221</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(0, 0, 0);</string>
              </property>
              <property name="text">
               <string>PushButton</string>
              </property>
             </widget>
            </widget>
            <widget class="QWidget" name="PgOffice">
             <widget class="QPushButton" name="pushButton_4">
              <property name="geometry">
               <rect>
                <x>130</x>
                <y>150</y>
                <width>291</width>
                <height>211</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(85, 0, 255);</string>
              </property>
              <property name="text">
               <string>PushButton</string>
              </property>
             </widget>
            </widget>
            <widget class="QWidget" name="PgSeguridad">
             <widget class="QPushButton" name="pushButton_5">
              <property name="geometry">
               <rect>
                <x>110</x>
                <y>140</y>
                <width>341</width>
                <height>211</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(255, 85, 0);</string>
              </property>
              <property name="text">
               <string>PushButton</string>
              </property>
             </widget>
            </widget>
            <widget class="QWidget" name="PgJuego">
             <widget class="QPushButton" name="pushButton_11">
              <property name="geometry">
               <rect>
                <x>100</x>
                <y>120</y>
                <width>361</width>
                <height>251</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(64, 131, 255);</string>
              </property>
              <property name="text">
               <string>PushButton</string>
              </property>
             </widget>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
